runners/flink/flink_runner.gradle

```diff
@@ -177,6 +177,7 @@ dependencies {
   // Required for embedded fn execution
   runtimeOnly project(":sdks:java:harness")
   if (flink_version.startsWith('1.10')) {
+      compile("com.lyft:streamingplatform-dryft-sdk:2.2-SNAPSHOT")
```

Comment:
We will need to look into separating the source from the dryft SDK to use it here.